indicadores/test_models.py

```diff
@@ -0,0 +1,144 @@
+from django.test import TestCase
+from familias.models import Familia
+from .models import Oficio, Periodo, Transaccion, Ingreso
+
+
+class TestOficio(TestCase):
+    """ Unit test suite for testing the Oficio model in .models
+
+    Attributes:
+    -----------
+    nombre : String
+        Stores the name that will be used for the name of the oficio object.
+    """
+
+    nombre = 'Electricista'
```

Comment:
It should be included as part of the setup of this tests, since they are initialization. This could be reviewed in this tests: https://github.com/erikiado/jp2_online/blob/develop/administracion/tests.py